src/main/java/io/github/ImpactDevelopment/installer/target/targets/Vanilla.java

```diff
@@ -174,6 +172,19 @@ public String apply() throws IOException {
         return "Impact has been successfully installed";
     }
 
+    @Override
+    public String installOptifine() throws IOException {
+        if (optifine == null) {
+            throw new IllegalStateException("No optifine specified, cannot install OptiFine");
+        }
+
+        Path libs = config.getSettingValue(MinecraftDirectorySetting.INSTANCE).resolve("libraries");
+        System.out.println("Installing OptiFine into "+libs.toString());
```

Comment:
autoformat ur code or i'll start crying

` + `

Reply:
This project doesn't have an agreed upon format? We talked about adding a gradle task but it never happened.

Reply:
the agreed upon format is always exactly the format that already exists in the codebase, on the branch that you are PR-ing into. 😡

Reply:
Fixed